### PR TITLE
Remove Custom Hook

### PR DESCRIPTION
**Pull request by AnnaViktorovna**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,37 +3,51 @@ import "./App.css";
...
         const filterTodo = todoList.filter((todo) => todo.id !== id);
         setTodoList(filterTodo);
-        inputRef.current.focus();
     }
 
     return (
         <>
             <div className="App">
                 <h1>My Todo List</h1>
                 <AddTodoForm onAddTodo={addTodo} />
```
Comment (gsvidal) [inline src/App.jsx]:
```suggestion
                <AddTodoForm onAddTodo={addTodo} inputRef={inputRef}/>
```

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { useState } from "react";
+import { useState} from "react";
 import InputWithLabel from "./assets/InputWithLabel";
 
 export default function AddTodoForm({ onAddTodo }) {
```
Comment (gsvidal) [inline src/AddTodoForm.jsx]:
```suggestion
export default function AddTodoForm({ onAddTodo, inputRef }) {
```

File: src/assets/InputWithLabel.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ export default function InputWithLabel({id, value, onChange, children}) {
 
     useEffect(() => {
         inputRef.current.focus()
-});
+}, );
```
Comment (gsvidal) [inline src/assets/InputWithLabel.jsx]:
```suggestion
}, []);
```

**Review by gsvidal — CHANGES_REQUESTED** [review]
You should make some fixes. I highlighted some of them here. Watch your Slack DM.

**Review by AnnaViktorovna — COMMENTED** [review]
changes after revier's suggestions

**Pull request merged**